README.md

```diff
@@ -15,21 +18,33 @@ including configuration for the menu.
 This is a Sushi project and can use HL7 IG Publisher to build locally:
 
   1. Clone this respository
-  2. Run `./scripts/_updatePublisher.sh` to get the latest IG publisher
-  3. Run `./scripts/_genonce.sh` to generate the IG
-  4. Run `open output/index.html` to view the IG website
+  1. Run `./scripts/_updatePublisher.sh` to get the latest IG publisher
+  1. Install `sushi` if you don't have it already with: `npm i fsh-sushi`
+  1. Run `./scripts/_genonce.sh` to generate the IG
+  1. Run `open output/index.html` to view the IG website
+      <details>
+        <summary>Instructions for viewing the IG in a local <code>http-server</code>...</summary>
+
+        ```sh
+        npm i http-server
```

Comment:
FWIW - there may be a more idiomatic way to do this with `nodemon`.  I just documented the first thing that worked for me.  :-)